web/src/components/tasks/RaffleBingoPaginator.test.tsx

```diff
@@ -0,0 +1,146 @@
+import { RaffleBingoPaginator } from "@/components/tasks/RaffleBingoPaginator";
+import { getMergedConfig } from "@/contexts/configContext";
+import { Task } from "@/lib/types/types";
+import { getTaskStatusUpdatedMessage } from "@/lib/utils/helpers";
+import { convertTaskMd } from "@/lib/utils/markdownReader";
+import { generateTask } from "@/test/factories";
+import { useMockRouter } from "@/test/mock/mockRouter";
+import { useMockRoadmap } from "@/test/mock/mockUseRoadmap";
+import { setupStatefulUserDataContext, WithStatefulUserData } from "@/test/mock/withStatefulUserData";
+import { generateUserData } from "@businessnjgovnavigator/shared/test";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+jest.mock("next/router", () => ({ useRouter: jest.fn() }));
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+jest.mock("@/lib/roadmap/buildUserRoadmap", () => ({ buildUserRoadmap: jest.fn() }));
+
+const Config = getMergedConfig();
+
+const fetchRaffleBingoTab = async (filename: string): Promise<Task> => {
+  /*
+    This function is a paired down version of the fetchTaskByFilename function to ensure we're getting the
+    real task in the context of a test environment.
+  */
+  const tabTask = await import(`@businessnjgovnavigator/content/roadmaps/raffle-bingo-steps/${filename}.md`);
+
+  return {
+    ...convertTaskMd(tabTask.default),
+    unlockedBy: [],
+    filename,
+  };
+};
+
+let step1: Task;
+let step2: Task;
+
+describe("<RaffleBingoPaginator />", () => {
+  beforeEach(async () => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+    useMockRouter({});
```

Comment:
[sand] These aren't doing anything. Having them in here isn't hurting, but it's also not helping.

We've been spending a lot of time talking about setting up tests correctly, specifically related to the data that we're providing via generators. This isn't that case, but it is related in my opinion. We should only be doing things that are necessary in the context of the test and including superfluous information makes it less clear what's important for the test.

Consider the purpose of the things we're including in the test. If it's not necessary to get things running, drop it. 
```suggestion
```